README.adoc

```diff
@@ -34,6 +34,7 @@ buildPlugin(
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `useContainerAgent` (default: `false`) - uses a link:https://github.com/jenkins-infra/documentation/blob/main/ci.adoc#container-agents[Container agent] instead of a Virtual Machine: usually faster to start and generates less costs for the project
 ** Please note that the implementation of "containers" can be changed over time
+* `forkCount` (default: null) - Execute tests with `forkCount` Java virtual machines. Common values include `0.45C`, `1`, `2`, and `1C`.
```

Comment:
What's the difference between `1` and `1C`?

Reply:
Found an explanation here: https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html

> if you terminate the value with a 'C', that value will be multiplied with the number of available CPU cores in your system

Reply:
The `1C` value forks a virtual machine per core on the machine.  If the machine has 16 cores, then 16 virtual machines are run in parallel.  `0.45C` is the value that @jglick uses on his macOS laptop with good results.  I use `1C` on the older separate machines that I use to run tests.

Reply:
Ahem, I use a ThinkPad with Linux. :grin: